interface/forms/eye_mag/a_issue.php

```diff
@@ -164,7 +182,7 @@
 
             if ($local =="1") { // leave FH/SocHx/ROS for later - done below separately
                 while ($res = sqlFetchArray($qry)) { //Should we take the top 10 and display alphabetically?
-                    echo " aopts['" .attr($key). "'][aopts['" .attr($key). "'].length] = new Option('".attr(trim($res['option_id']))."', '".attr(xl_list_label(trim($res['title'])))."', false, false);\n";
+                    echo " aopts['" .attr($key). "'][aopts['" .attr($key). "'].length] = new Option('".attr(xl_list_label(trim($res['title'])))."', '".attr(trim($res['option_id']))."', false, false);\n";
```

Comment:
For above line, can do the following to make it more solid since this is within javascript context:
```
echo " aopts['" .attr($key). "'][aopts['" .attr($key). "'].length] = new Option(".js_escape(xl_list_label(trim($res['title']))).", ".js_escape(trim($res['option_id'])).", false, false);\n";
```
(note that I used the js_escape and removed the single quotes surrounding them)